.github/workflows/deploy.yml

```
@@ -3,18 +3,22 @@ name: Deploy to EC2 (Blue-Green)
 jobs:
   deploy:
     runs-on: ubuntu-latest
 
     env:
       SLACK_DEV_WEBHOOK_URL: ${{ secrets.SLACK_DEV_WEBHOOK_URL }}
       SLACK_PROD_WEBHOOK_URL: ${{ secrets.SLACK_PROD_WEBHOOK_URL }}
+      SERVICES: "gateway-service help-service notification-service review-service point-service user-service config-server eureka-server"
 
     steps:
-    - uses: actions/checkout@v4
+    - name: Checkout
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)